testing/tools/dagon/Makefile

```diff
@@ -11,16 +11,16 @@ $(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
 TEST_TARGET := false
 
 # uncomment to disable generate pony related targets (build/test/clean) for pony sources in this directory
-#PONY_TARGET := false
+PONY_TARGET := false
 
 # uncomment to disable generate exs related targets (build/test/clean) for elixir sources in this directory
-#EXS_TARGET := false
+EXS_TARGET := false
 
 # uncomment to disable generate docker related targets (build/push) for Dockerfile in this directory
-#DOCKER_TARGET := false
+DOCKER_TARGET := false
 
 # uncomment to disable generate recursing into Makefiles of subdirectories
-#RECURSE_SUBMAKEFILES := false
+RECURSE_SUBMAKEFILES := false
```

Comment:
Are we intentionally disabling recursing into sub-makefiles for the dagon utilities because dagon is to be removed? If yes, why are we changing `dagon.pony` at all? If no, then why aren't we changing the dagon utilities?

Reply:
As per #1603, I'm going to remove Dagon altogether, and since this switch means we have to make a decision between maintaining Dagon, isolating it, or removing it, I'm going to remove it as part of this PR.